src/webapp/azext_webapp/create_util.py

```diff
@@ -17,7 +17,9 @@
     DOTNET_RUNTIME_NAME,
     DOTNET_VERSION_DEFAULT,
     DOTNET_VERSIONS,
-    STATIC_RUNTIME_NAME)
+    STATIC_RUNTIME_NAME,
+    PYTHON_RUNTIME_NAME,
+    PYTHON_VETSION_DEFAULT)
```

Comment:
typo, i think you mean VERSION?

Reply:
Thanks for catching this. Will update this.